Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Account.yaml

```diff
@@ -25,6 +25,24 @@ apis:
         type: VerifyAccountReq
       response:
         type: APISuccess
+  - PUT:
+      endpoint: /updateRole/{personId}/{roleId}
+      auth: ApiAuthV2
+      params:
+        personId: Id Person
+        roleId: Id Role
+      response:
+        type: APISuccess
+      helperApi:
+        - PUT:
+            endpoint: /updateRole/{personId}/{roleId}/{accessType}
```

Comment:
`roleId` not required in helper api, because `roleId` is different on bpp and dashboard side
`accessType` is enough